transports/azure-service-bus/configuration.md

```diff
@@ -39,3 +39,4 @@ These settings control how the transport connects to the broker.
  * `UseWebSockets()`: Configures the transport to use AMQP over websockets.
  * `TimeToWaitBeforeTriggeringCircuitBreaker(TimeSpan)`: The time to wait before triggering the circuit breaker after a critical error occurred. Defaults to 2 minutes.
  * `CustomTokenProvider(ITokenProvider)`: Allows replacement of the default token provider, which uses the shared secret in the connection string for authentication. This opens up additional authentication mechanisms such as [shared access signatures](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas), [SAML, Oauth, SWT, windows authentication](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicebus.tokenprovider?view=azure-dotnet), [managed identities for Azure resources](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-managed-service-identity), or even custom implementations.
+ partial: custom-retry-policy
```

Comment:
Recommend a buffer line, otherwise I don't know how it's interpreted
```suggestion

 partial: custom-retry-policy
```

Reply:
I think it is supposed to be a part of this bulleted list?

Reply:
So this change was needed. There's no need to have a space between partial and the previous lines. Good to know 🙂